src/dispatch.rs

```diff
@@ -169,10 +171,12 @@ impl<'t> Dispatcher<'t> {
 
         #[cfg(target_os = "emscripten")]
         self.dispatch_seq(res);
+
+        self.dispatch_thread_local(res);
```

Comment:
It will do, but my worry is naming. Executing thread local systems in method named `wait` is 
 just implicit.